### PR TITLE
WELD-1898 Add overview graph and few improvements around bean archives

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

**Comment by weld-tester** [comment]
Triggering build using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 583 is now running using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/583

**Comment by weld-tester** [comment]
Build 583 outcome was FAILURE using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/583
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/583

**Comment by mkouba** [comment]
This is probably blocked by https://github.com/weld/core/pull/902.

**Comment by weld-tester** [comment]
Build 583 outcome was FAILURE using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/583
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/583

**Comment by weld-tester** [comment]
Build 583 outcome was FAILURE using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/583
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/583

**Comment by jharting** [comment]
retest this please

**Comment by weld-tester** [comment]
Triggering build using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 585 is now running using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/585

**Comment by weld-tester** [comment]
Build 585 outcome was SUCCESS using a merge of b83a663337be08e6a7b2d4e8821f2080d10f67e6 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/585
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/585

File: probe/core/src/main/java/org/jboss/weld/probe/Strings.java [hunk 1]
```diff
@@ -95,6 +95,7 @@
     public static final String LAST_PAGE = "lastPage";
     public static final String DATA = "data";
     public static final String FILTERS = "filters";
+    public static final String REPRESENATION = "representation";
```
Comment (jharting) [inline probe/core/src/main/java/org/jboss/weld/probe/Strings.java]: typo

**Comment by weld-tester** [comment]
Triggering build using a merge of d30cd58122b30f2ac0abd1fdcdf8cceb324b9faa on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 587 is now running using a merge of d30cd58122b30f2ac0abd1fdcdf8cceb324b9faa on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/587

**Comment by weld-tester** [comment]
Build 587 outcome was SUCCESS using a merge of d30cd58122b30f2ac0abd1fdcdf8cceb324b9faa on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/587
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/587

**Comment by mkouba** [comment]
I've just found a little bug in bean detail.

**Comment by weld-tester** [comment]
Triggering build using a merge of ab1f7520a6a37d405cb8926ed6aaf7abe72645e5 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 588 is now running using a merge of ab1f7520a6a37d405cb8926ed6aaf7abe72645e5 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/588

**Comment by weld-tester** [comment]
Build 588 outcome was SUCCESS using a merge of ab1f7520a6a37d405cb8926ed6aaf7abe72645e5 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/588
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/588

**Comment by jharting** [comment]
Merged, thanks!

**Pull request closed without merging**